src/delagent/agent/delagent.c

```diff
@@ -31,11 +32,59 @@ char BuildVersion[]="delagent build version: NULL.\n";
 #endif
 
 
+/***********************************************
+ Usage():
```

Comment:
I would prefer all function names and variable names to be start with lower case. It should be uniform at least.